sympy/core/numbers.py

```diff
@@ -3311,6 +3311,10 @@ def _sage_(self):
         import sage.all as sage
         return sage.golden_ratio
 
+    def _eval_rewrite_as_sqrt(self):
+        from sympy import sqrt
+        return S.Half + S.Half*sqrt(5)
```

Comment:
You can replace the above lines of code with just ``_eval_rewrite_as_sqrt = _eval_expand_func``.